pkg/agent/datamodel/azenvtypes.go

```diff
@@ -113,30 +113,6 @@ var (
 		ImageVersion:   "latest",
 	}
 
-	//Ubuntu1804Gen2OSImageConfig is Gen2 flavor the Ubunutu 18.04-LTS Linux distribution.
-	Ubuntu1804Gen2OSImageConfig = AzureOSImageConfig{
-		ImageOffer:     "aks",
-		ImageSku:       "aks-ubuntu-1804-gen2-2020-q3",
-		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "2020.08.06",
-	}
```

Comment:
we need this.

Reply:
I'll abandon this PR if we still need the code, but before I do it, where are we using it in AgentBaker?

Reply:
No I don't think we need the OSImageConfig map, but **if we keep it**, we should keep this Ubuntu1804Gen2OSImageConfig along with other four that you keep. It's for completeness.

If you remove the whole OSImageConfig map and all the image configs, I think that will also work

Reply:
I removed the whole "OSImageConfig" map since AgentBaker doesn't need it.